### ISSUE TITLE
Error in shinyMethyl::orderByName(shinyMethylSet2)

### ISSUE THREAD

**Issue by biga94** [Jfortin1/shinyMethyl#9]
Hi all,

as I try to run `runShinyMethyl(myShinyMethylSet)` with a previously created RGSet with minfi package, this error comes out:

`> runShinyMethyl(myShinyMethylSet)
Errore in shinyMethyl::orderByName(shinyMethylSet2) : 
  is(object, "shinyMethylSet") non è TRUE`

Of course, the object is a shinyMethylSet. I'm working on EPICv1 data

Best,

DB

**Comment by cewim** [comment]
I'm suddenly having the same issue using runShinyMethyl() with EPICv1 data. @biga94, did you ever figure this out?

```
> summarized_set <- shinySummarize(RGset_1)
[shinySummarize] Extracting Red and Green channels 
[shinySummarize] Raw preprocessing 
[shinySummarize] Mapping to genome 
[shinySummarize] Computing quantiles 
[shinySummarize] Computing principal components 

> summarized_set
 Object:  shinyMethylSet 
 Number of samples:  97 
 Phenotype:  12 covariates 
 Origin object:  RGChannelSet 
 Array:  IlluminaHumanMethylationEPIC 

> runShinyMethyl(summarized_set)
Error in shinyMethyl::orderByName(shinyMethylSet2) : 
  is(object, "shinyMethylSet") is not TRUE

> runShinyMethyl(summarized_set,shinyMethylSet2 = NULL)
Error in shinyMethyl::orderByName(shinyMethylSet2) : 
  is(object, "shinyMethylSet") is not TRUE
```

sessionInfo()
```
> sessionInfo()
R version 4.4.0 (2024-04-24 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils    
[7] datasets  methods   base     

other attached packages:
 [1] methylclock_1.10.0                                 
 [2] quadprog_1.5-8                                     
 [3] devtools_2.4.5                                     
 [4] usethis_2.2.3                                      
 [5] methylclockData_1.12.0                             
 [6] sesame_1.22.1                                      
 [7] sesameData_1.22.0                                  
 [8] FlowSorted.BloodExtended.EPIC_1.1.1                
 [9] FlowSorted.Blood.EPIC_2.8.0                        
[10] IlluminaHumanMethylationEPICanno.ilm10b4.hg19_0.6.0
[11] IlluminaHumanMethylationEPICmanifest_0.3.0         
[12] ExperimentHub_2.12.0                               
[13] AnnotationHub_3.12.0                               
[14] BiocFileCache_2.12.0                               
[15] dbplyr_2.5.0                                       
[16] data.table_1.15.4                                  
[17] lubridate_1.9.3                                    
[18] forcats_1.0.0                                      
[19] stringr_1.5.1                                      
[20] dplyr_1.1.4                                        
[21] purrr_1.0.2                                        
[22] readr_2.1.5                                        
[23] tidyr_1.3.1                                        
[24] tibble_3.2.1                                       
[25] ggplot2_3.5.1                                      
[26] tidyverse_2.0.0                                    
[27] shinyMethyl_1.40.0                                 
[28] minfi_1.50.0                                       
[29] bumphunter_1.46.0                                  
[30] locfit_1.5-9.9                                     
[31] iterators_1.0.14                                   
[32] foreach_1.5.2                                      
[33] Biostrings_2.72.0                                  
[34] XVector_0.44.0                                     
[35] SummarizedExperiment_1.34.0                        
[36] Biobase_2.64.0                                     
[37] MatrixGenerics_1.16.0                              
[38] matrixStats_1.3.0                                  
[39] GenomicRanges_1.56.0                               
[40] GenomeInfoDb_1.40.1                                
[41] IRanges_2.38.0                                     
[42] S4Vectors_0.42.0                                   
[43] BiocGenerics_0.50.0                                
[44] futile.logger_1.4.3                                

loaded via a namespace (and not attached):
  [1] fs_1.6.4                   bitops_1.0-7              
  [3] httr_1.4.7                 RColorBrewer_1.1-3        
  [5] dynamicTreeCut_1.63-1      backports_1.4.1           
  [7] profvis_0.3.8              tools_4.4.0               
  [9] doRNG_1.8.6                utf8_1.2.4                
 [11] R6_2.5.1                   HDF5Array_1.32.0          
 [13] rhdf5filters_1.16.0        urlchecker_1.0.1          
 [15] withr_3.0.0                gridExtra_2.3             
 [17] prettyunits_1.2.0          base64_2.0.1              
 [19] preprocessCore_1.66.0      quantreg_5.98             
 [21] cli_3.6.2                  pacman_0.5.1              
 [23] formatR_1.14               AnnotationHubData_1.34.0  
 [25] genefilter_1.86.0          askpass_1.2.0             
 [27] Rsamtools_2.20.0           txdbmaker_1.0.1           
 [29] siggenes_1.78.0            illuminaio_0.46.0         
 [31] stringdist_0.9.12          AnnotationForge_1.46.0    
 [33] sessioninfo_1.2.2          scrime_1.3.5              
 [35] limma_3.60.0               readxl_1.4.3              
 [37] impute_1.78.0              rstudioapi_0.16.0         
 [39] RSQLite_2.3.6              generics_0.1.3            
 [41] BiocIO_1.14.0              car_3.1-2                 
 [43] Matrix_1.7-0               fansi_1.0.6               
 [45] abind_1.4-5                lifecycle_1.0.4           
 [47] yaml_2.3.8                 ggpmisc_0.6.0             
 [49] carData_3.0-5              rhdf5_2.48.0              
 [51] biocViews_1.72.0           SparseArray_1.4.3         
 [53] grid_4.4.0                 blob_1.2.4                
 [55] promises_1.3.0             crayon_1.5.3              
 [57] miniUI_0.1.1.1             lattice_0.22-6            
 [59] GenomicFeatures_1.56.0     annotate_1.82.0           
 [61] KEGGREST_1.44.1            pillar_1.9.0              
 [63] knitr_1.48                 beanplot_1.3.1            
 [65] rjson_0.2.21               codetools_0.2-20          
 [67] glue_1.7.0                 ExperimentHubData_1.30.0  
 [69] remotes_2.5.0              vctrs_0.6.5               
 [71] png_0.1-8                  cellranger_1.1.0          
 [73] gtable_0.3.5               ggpp_0.5.8-1              
 [75] cachem_1.0.8               xfun_0.44                 
 [77] S4Arrays_1.4.0             mime_0.12                 
 [79] survival_3.6-4             statmod_1.5.0             
 [81] ellipsis_0.3.2             nlme_3.1-164              
 [83] xts_0.13.2                 bit64_4.0.5               
 [85] progress_1.2.3             filelock_1.0.3            
 [87] nor1mix_1.3-3              colorspace_2.1-0          
 [89] DBI_1.2.3                  tidyselect_1.2.1          
 [91] bit_4.0.5                  compiler_4.4.0            
 [93] curl_5.2.1                 httr2_1.0.1               
 [95] graph_1.82.0               BiocCheck_1.40.0          
 [97] SparseM_1.83               xml2_1.3.6                
 [99] RPMM_1.25                  DelayedArray_0.30.0       
[101] rtracklayer_1.64.0         scales_1.3.0              
[103] RBGL_1.80.0                rappdirs_0.3.3            
[105] digest_0.6.35              rmarkdown_2.27            
[107] GEOquery_2.72.0            htmltools_0.5.8.1         
[109] pkgconfig_2.0.3            sparseMatrixStats_1.16.0  
[111] fastmap_1.1.1              rlang_1.1.3               
[113] htmlwidgets_1.6.4          UCSC.utils_1.0.0          
[115] shiny_1.8.1.1              DelayedMatrixStats_1.26.0 
[117] zoo_1.8-12                 jsonlite_1.8.8            
[119] BiocParallel_1.38.0        mclust_6.1.1              
[121] wheatmap_0.2.0             RCurl_1.98-1.14           
[123] magrittr_2.0.3             polynom_1.4-1             
[125] GenomeInfoDbData_1.2.12    Rhdf5lib_1.26.0           
[127] munsell_0.5.1              Rcpp_1.0.12               
[129] stringi_1.8.4              PerformanceAnalytics_2.0.4
[131] zlibbioc_1.50.0            MASS_7.3-60.2             
[133] plyr_1.8.9                 pkgbuild_1.4.4            
[135] splines_4.4.0              multtest_2.60.0           
[137] hms_1.1.3                  ggpubr_0.6.0              
[139] RUnit_0.4.33               ggsignif_0.6.4            
[141] rngtools_1.5.2             reshape2_1.4.4            
[143] biomaRt_2.60.1             pkgload_1.4.0             
[145] futile.options_1.0.1       BiocVersion_3.19.1        
[147] XML_3.99-0.16.1            evaluate_0.24.0           
[149] lambda.r_1.2.4             BiocManager_1.30.23       
[151] tzdb_0.4.0                 httpuv_1.6.15             
[153] MatrixModels_0.5-3         openssl_2.1.2             
[155] reshape_0.8.9              BiocBaseUtils_1.6.0       
[157] broom_1.0.6                xtable_1.8-4              
[159] restfulr_0.0.15            rstatix_0.7.2             
[161] later_1.3.2                OrganismDbi_1.46.0        
[163] memoise_2.0.1              AnnotationDbi_1.66.0      
[165] GenomicAlignments_1.40.0   cluster_2.1.6             
[167] writexl_1.5.0              timechange_0.3.0
```

**Comment by Jfortin1** [comment]
Thanks @biga94 and @cewim for reporting the bug -- it is now fixed in the devel branch (v1.41.1)
